web/src/index.tsx

```diff
@@ -1,9 +1,8 @@
-import "@utils/AssetPath";
-
 import React from "react";
 
 import ReactDOM from "react-dom";
 
+import "@utils/AssetPath";
```

Comment:
Did the linter move this or was this manually done?
I believe the AssetPath import needs to be the very first import to avoid issues with webpack due to our dynamic pathing.

Reply:
I had some errors raised by the transpiler so I moved imports around until I fixed it. Would not we see any issue in the integration tests if it was broken? If not, I should probably cover that with a test.

Reply:
I just double checked it's no problems as long as it's imported where it is right now, if it's imported after `import App from "@root/App";` it will cause issues.

Reply:
Also the specific issue we see is that on `RegisterOneTimePassword.tsx` we see broken links for the AppStoreBadges:

![Screenshot_2021-07-14-06-52-09](https://user-images.githubusercontent.com/3339418/125593517-4ea568f5-c558-4ee1-955b-7e8fdd9a040f.png)